internal_ws/ykcompile/src/stack_builder.rs

```diff
@@ -10,10 +12,10 @@ pub(crate) struct StackBuilder {
 
 /// A naive stack builder for allocating objects on the stack.
 ///
-/// Naive because it could allocate less space by caching up the requested allocations and
+/// XXX I can't parse this sentence. Naive because it could allocate less space by caching up the requested allocations and
 /// better-packing them.
 ///
-/// The top of the stack, before any calls to `alloc()` is assumed to be appropriately aligned.
+/// XXX or this sentence. The top of the stack, before any calls to `alloc()` is assumed to be appropriately aligned.
```

Comment:
It means the stack must be aligned before you allocate using our stack builder. If the stack is misaligned and you use `StackBuilder` the stack will still be misaligned.

Reply:
Fixed in 114a1ca8742ee17f5b783c70a197e6a5d2a7d72e.